shared/utils.js

```diff
@@ -70,7 +70,7 @@ function setLogLevel(logLevel) {
  */
 function getVersionNumberFromString(versionString) {
 	// Only supporting major/minor version checking at this point.
-    var versionRegex = new RegExp(/^[^\d]*(\d+)(\.(\d+))(\.(\d+))?.*$/, 'm');
+    var versionRegex = new RegExp(/^[^\d]*(\d+)(\.(\d+))?(\.(\d+))?/, 'm');
```

Comment:
The extra ? is so that we can handle a version that is just a number.
I dropped the .*$ to handle version that contains more that x.y.z-zzz for instance sfdx -v returns:
`sfdx-cli/7.63.0 darwin-x64 node-v10.21.0`